package.json

```diff
@@ -9,10 +9,10 @@
   "license": "MIT",
   "scripts": {
     "start": "node node_modules/react-native/local-cli/cli.js start",
-    "test": "yarn test:eslint && yarn test:flow && yarn test:js",
+    "test": "yarn test:eslint && yarn test:flow && yarn test:jest",
     "test:eslint": "eslint 'js/**/*.js' 'example/**/*.js'",
     "test:flow": "flow check",
-    "test:js": "echo 0"
+    "test:jest": "jest js/"
```

Comment:
you can probably strip the "js/" unless you have other test files you don't want Jest to run.